### ISSUE TITLE
Polkadot Ledger integration

### ISSUE THREAD

**Issue by faboweb** [luniehq/lunie#3849]
Library by Juan: https://github.com/Zondax/ledger-kusama

**Comment by Bitcoinera** [comment]
Yeey! So exciting! 🤩

**Comment by Bitcoinera** [comment]
yeah, this is definitely stale. As long as we don't get the Polkadot app on the Ledger Live store 🤷 Not so much to do, sadly

**Comment by Bitcoinera** [comment]
Hey, I have some updates on this one! One friend from La Colmena managed to do this for Polkastats and has just explained to me "the way".

So I'll be happy to do this at the extent it is possible nowadays... Only that currently [the Ledger app repo](https://github.com/Zondax/ledger-kusama) and [the web library one](https://github.com/Zondax/ledger-polkadot-js) are currently not compatible 🤷 So it still doesn't work

**Comment by faboweb** [comment]
So let's wait until they are compatible?

**Comment by Bitcoinera** [comment]
Waiting on the Ledger app release

**Comment by jbibla** [comment]
if and when the app is released we'll still need to apply for a grant. we can close for now.

**Comment by Bitcoinera** [comment]
XD
